README.md

```diff
@@ -9,6 +10,7 @@ It follows the word orders in sentence, same as the words order in array column
 <a href="https://www.buymeacoffee.com/arshadkazmi42" target="_blank"><img src="https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png" alt="Buy Me A Coffee" style="height: auto !important;width: auto !important;" ></a>
 <a href="https://www.patreon.com/bePatron?u=15454240" target="_blank"><img src="https://c5.patreon.com/external/logo/become_a_patron_button.png" alt="Become a Patron!" height="40"></a>
 
+
```

Comment:
There is a extra line break here.
Can you remove it pls

Reply:
Line break removed.
```suggestion

```